.github/workflows/Pipeline.yml

```diff
@@ -135,7 +136,8 @@ jobs:
       - name: Check Static Typing
         continue-on-error: true
         run: |
-          mypy --html-report htmlmypy --pretty pyEDAA/ProjectModel
+          pwd
```

Comment:
This is probably a leftover from some testing?

Reply:
Currently mypy gives still some problems.  
This is to be fixed in a next release.